db/00000000000006_optional_task_expiration.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE tasks ALTER COLUMN task_expiration DROP NOT NULL;
```

Comment:
Is there any data that needs to be migrated, replacing MAX with null?

Reply:
Nope. Our current stance is that we don't need to provide data migrations as there is no data to migrate -- we will need to change this stance once we have a deployed environment with data we care to keep.